### PR TITLE
Modal styling

### PR DESCRIPTION
**Pull request by BenAlderfer**
Fixes issues #51, #56, and #61.

- Cancel button turned into x in corner
- Save and delete buttons grow to half width
- Notes textarea added 
- Add pre-req button styling fixed
- Fixed input styling so adjacent inputs appear as one continuous section
- Pre-req delete button mirrors semester delete button
- Also updated semester wording to term

modal

### CODE REVIEW

**Comment by khanny17** [comment]
Looks good!

My recommendations:
- Add a bold heading to the notes field just like how the Name, Department, Number, and the other fields have above them, for consistency sake.
- Make the notes textfield _not_ resizable
- Make the X bigger and aligned with the Edit Course title. I picture it being the same size as the letters in the title

**Comment by sklei2** [comment]
Couple comments/questions:
 - As @khanny17 said, bring the x in the top right down to line up with the "Edit Course" at the top. And don't have the text be resizeable, also can we make it a little taller? Like 70px (which I think is 5em, if my calculations with the 14px font is right) so that we can see 3 lines of text at any one point instead of 2?
 - Do we want the delete 'x' for prerequisites to change color on hover over the entire prerequisite block? I feel like it is a little strange that it does that. Cause it kinda takes my eye away from the class title I may try to read and make we confused as to why it is active. I think changing color when I hover over the X itself is good enough.
 - Is there a _max_ number of prerequisites that we'll allow a user to add? Cause if that number gets a little too big, we'll have a really tall modal.  Maybe have it be able to show around 3 at once without needing to scroll. Since I can't imagine a class with more than 3 prerequisites in most cases.

image

**Comment by BenAlderfer** [comment]
Right now I've fixed/added:

- x coloring for deleting pre-reqs to just on x hover
- Notes header
- Notes fixed height of 3 lines
- Close x is larger and aligned with modal title
- Added some spacing between modal buttons

modal

Haven't touched the number of pre-reqs since I'm not sure about that yet.

**Comment by khanny17** [comment]
Don't worry about the number of prereqs. We can worry about limiting size of plans (including prereq number) later. For now if a user wants to put 50 prereqs, be my guest.

File: public/js/directives/course/course-edit-modal.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <div class="course-edit-modal">
     <div class="modal-header">
+        <button type="button" class="close" ng-click="cancel()" aria-label="Close"><span aria-hidden="true">&times;</span></button>
```
Comment (khanny17) [inline public/js/directives/course/course-edit-modal.html:3]: You could use the fa icon fa-times instead of this, if you prefer. Or even just the letter X

**Review by khanny17 — APPROVED** [review]

File: public/js/directives/course/course-edit-modal.html (hunk 1 above)
Comment (BenAlderfer) [reply public/js/directives/course/course-edit-modal.html:3]: I just did it that way to be consistent with the other modals

**Pull request merged**